app/actions/index.ts

```diff
@@ -50,4 +51,7 @@ export function handleContentTypeChange(value:string, requestId:number){
 }
 export function handlePaginationChange(value: string, tabId:number, field:string) {
   return {type: PAGINATION_CHANGE, payload: {field, value, tabId}};
+}
+export function sendSuccessMessage(tabId:number){
```

Comment:
This should accept message as well

Reply:
As discussed there is no need for tabId. this should not depend on tabId. anyway we will be clear this on mounting.